lisa/notifiers/json.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import json
+import os
+from dataclasses import dataclass
+from datetime import datetime
+from decimal import Decimal
+from enum import Enum
+from typing import Any, List, Type, cast
+
+from dataclasses_json import dataclass_json
+
+from lisa import constants, messages, notifier, schema
+
+
+@dataclass_json()
+@dataclass
+class JsonSchema(schema.Notifier):
+    path: str = "lisa.json"
+
+
+class Json(notifier.Notifier):
```

Comment:
This class is specified to perf message, so the name shouldn't be called like `Json`, use `PerfDump`.

Reply:
updated